src/EdpGithub/Collection/RepositoryCollection.php

```diff
@@ -125,7 +126,7 @@ public function add($offset, $element)
         $this->elements[$offset] = $element;
     }
 
-    private function getPagination($response)
+    private function getPagination(Http\Response $response)
```

Comment:
Yes! Maybe this is another BC break because change the signature of method.. In my opinion is a good enhancement but I attend @EvanDotPro for this merge :)